pkg/apis/camel/v1alpha1/types.go

```diff
@@ -135,6 +136,12 @@ type IntegrationContextStatus struct {
 	Digest string                  `json:"digest,omitempty"`
 }
 
+// ClasspathEntry --
+type ClasspathEntry struct {
+	ID       string `json:"id" yaml:"id"`
+	Location string `json:"location,omitempty" yaml:"location,omitempty"`
```

Comment:
We store both the ID and the relative location?

Reply:
Or that is intended to be the location in the m2 repo in the builder pod? (or both?)